HPXMLtoOpenStudio/resources/hpxml.rb

```diff
@@ -270,6 +270,10 @@ class HPXML < Object
   PVTrackingType1Axis = '1-axis'
   PVTrackingType1AxisBacktracked = '1-axis backtracked'
   PVTrackingType2Axis = '2-axis'
+  RadiantBarrierLocationUndersideOfRafters = 'underside of rafters'
```

Comment:
How is "underside of rafters" different than "top side of truss under sheathing"?

Reply:
Green line Vs Orange line in this image? 
![image](https://github.com/NREL/OpenStudio-HPXML/assets/45133086/f58b5d0e-5310-4825-8eb7-b935485a3ff5)

Reply:
(It didn't use to be an HPXML enumeration choice but a [HPXML user requested it](https://github.com/hpxmlwg/hpxml/issues/16).)